components/admin_console/billing/plan_details/feature_list.tsx

```diff
@@ -41,11 +41,7 @@ const FeatureList = (props: FeatureListProps) => {
         intl.formatMessage(
             {
                 id: 'admin.billing.subscription.planDetails.features.limitedBoardCards',
-                defaultMessage: 'Limited to {limit} board cards per workspace',
```

Comment:
Current master branch webapp code will show this even after removing the limit in Stripe.